src/main/java/org/junit/runners/Parameterized.java

```diff
@@ -67,45 +88,58 @@
  * <dd>...</dd>
  * </dl>
  * <p>
- * In the example given above, the <code>Parameterized</code> runner creates
- * names like <code>[1: fib(3)=2]</code>. If you don't use the name parameter,
- * then the current parameter index is used as name.
+ * In the example given above, the <code>Parameterized</code> annotation
```

Comment:
I personally find the previous wording of this section easier to read and I think it provides enough information for people to follow.